packages/visual-editor/src/hooks/useComponent.tsx

```diff
@@ -98,7 +98,7 @@ export const useComponent = ({
 
     const element = React.createElement(
       ImportedComponentErrorBoundary,
-      null,
+      { componentID: node.data.blockId },
```

Comment:
nitpick:
```suggestion
      { componentId: node.data.blockId },
```

Reply:
Until today, I believed that I wanted to write acronyms & abbreviations with a casing like `MyAPIClass`, `apiFunction`, `IOFunction`, or `HTMLParser` but today I understood after a little research that your suggested casing not only has some advantages in general, also our whole SDK codebase follows this pattern. So will adjust it! :)